README.md

```diff
@@ -1 +1,39 @@
-# portfolio-code-review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
Fantastic descriptive README!

Reply:
Awesome job pointing out the highlights of the project! :clap: :100: